crates/oxc_linter/src/snapshots/no_duplicates.snap

```diff
@@ -540,14 +540,6 @@ source: crates/oxc_linter/src/tester.rs
    ╰────
   help: Merge these imports into a single import statement
 
-  ⚠ eslint-plugin-import(no-duplicates): Module './foo' is imported more than once in this file
-   ╭─[index.ts:1:38]
- 1 │ import {AValue, type x, BValue} from './foo'; import {type y} from './foo'
```

Comment:
What happened here?

Reply:
It looks like maybe we were reporting the same line twice? It looks to be the same message as the one below?